components/monitoring/logging/base/configure-logging.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  collection:
+    logs:
+      fluentd: {}
+      type: fluentd
+---
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
```

Comment:
I think that this should be on its own file in a dedicated directory (base/cluster-log-forwarder/cloudwatch.yaml), and include it in the resources of the staging kustomization.
This will allow us to deploy the logging operator in dev environments without configuring forwarding.

Reply:
moving to another file makes sense. but we are using logging operator only for logforwarding here. we are not using the elasticsearch and kibana.